### PR TITLE
Added ui:step property support to uiSchema.

### PR DESCRIPTION
**Pull request by michal-damiecki**
UIE-207

### CODE REVIEW

File: src/Dialog/formComponents/fields/StringField.js [hunk 1]
```diff
@@ -79,7 +80,10 @@ function StringField(props) {
     const Widget = getWidget(schema, widget, widgets);
     return <Widget {...commonProps} placeholder={placeholder} />;
   }
-  return <TextWidget {...commonProps} placeholder={placeholder} />;
+  return <TextWidget {...commonProps}
+    placeholder={placeholder}
+    step={step}
```
Comment (stepek) [inline src/Dialog/formComponents/fields/StringField.js]: Please move `step` property to `commonProps`, because nice to have this props in other widgets.

**Comment by michal-damiecki** [comment]
@stepek Done

File: src/components/forms/Input.js [hunk 2]
```diff
@@ -8,7 +8,8 @@ export const Input = (
     onChange = () => {},
     type = 'text',
     placeholder = '',
-    isInvalid = false
+    isInvalid = false,
+    step
```
Comment (stepek) [inline src/components/forms/Input.js]: Please add default value and add propTypes.

File: src/Dialog/formComponents/widgets/BaseInput.js [hunk 3]
```diff
@@ -53,6 +53,7 @@ class BaseInput extends Component {
       options,      // eslint-disable-line
       schema,       // eslint-disable-line
       formContext,  // eslint-disable-line
+      step,
```
Comment (stepek) [inline src/Dialog/formComponents/widgets/BaseInput.js]: Please add PropTypes for `step`

**Comment by michal-damiecki** [comment]
@stepek Done

**Pull request merged**